UMA-Bots/Receiving-Chain-Relay-Dispute/src/network.ts

```diff
@@ -0,0 +1,8 @@
+export interface NetworkDataInterface {
+  hubPoolAddr: string;
+}
+
+export const NM_DATA: Record<number, NetworkDataInterface> = {
+  1: { hubPoolAddr: "0xc186fa914353c44b2e33ebe05f21846f1048beda" },
+  5: { hubPoolAddr: "0xBc079C669989C432aBd026956f31CF82C8400faa" }, //PoC
```

Comment:
in addition to Vasilis' comment, it would be nice if you use Network from forta-agent

```suggestion
import { Network } from "forta-agent";

export interface NetworkDataInterface {
  hubPoolAddr: string;
}

export const NM_DATA: Record<number, NetworkDataInterface> = {
  [Network.MAINNET]: { hubPoolAddr: "0xc186fa914353c44b2e33ebe05f21846f1048beda" },
  [Network.GOERLI]: { hubPoolAddr: "0xBc079C669989C432aBd026956f31CF82C8400faa" }, //PoC
};

```